### PR TITLE
Removed icon_folder

### PR DESCRIPTION
**Pull request by 2shady4u**
### Checklist for this pull request

- [X] I have read the [guidelines for contributing](../CONTRIBUTING.md).
- [X] I have checked that my code additions did not fail tests.

### Description

Building a project that uses inkgd with the help of [godot-ci](https://github.com/aBARICHELLO/godot-ci) gives several errors because of the icon_folder.SVG which, for some reason, cannot by loaded.

I don't know the exact cause of these errors, but the icon_folder can actually be removed since the editor folder icon can be accessed directly. (By using `get_icon("Folder", "EditorIcons")`)

DON'T MERGE YET, STILL HAVE TO VERIFY CI BEHAVIOUR AND DO ADDITIONAL TESTS!

### CODE REVIEW

**Comment by ephread** [comment]
> but the icon_folder can actually be removed since the editor folder icon can be accessed directly. (By using get_icon("Folder", "EditorIcons"))

Amazing, I didn't know about this. That's clearly the way to go, thanks!

> DON'T MERGE YET, STILL HAVE TO VERIFY CI BEHAVIOUR AND DO ADDITIONAL TESTS!

Sure thing, I've marked this a WIP.

**Comment by 2shady4u** [comment]
Hey @ephread !

Sorry, I seem to have forgotten about the existence of this PR :/
I've been using the same tactic (using the edit folder icon) in one of my other repositories and haven't encountered any errors.

Also for completeness' sake here's a project that enables the user to browse and search for other
editor icons: https://github.com/pycbouh/godot-editor-theme-explorer

TLDR: You can safely merge this if you want.

**Comment by ephread** [comment]
> Sorry, I seem to have forgotten about the existence of this PR :/

No worries :)

> Also for completeness' sake here's a project that enables the user to browse and search for other
editor icons: https://github.com/pycbouh/godot-editor-theme-explorer

Great stuff, they'll get a star. ⭐

> TLDR: You can safely merge this if you want.

Perfect!
